.github/ISSUE_TEMPLATE/feature-proposal--developer-.md

```diff
@@ -33,34 +33,34 @@ assignees: ''
 >
 > 1. NFR: <FURPS+ Category>: <Title>: <Description>
 
----
 ## Analysis
 
 ### Analysis Object Model 
 > What are the involved Analysis Objects? 
 
 ### Dynamic Behavior 
-> Include dynamic models (Activity Diagram, State Chart Diagram, Communication Diagram) here to outline the dynamic nature of the PROBLEM 
+> Include dynamic models (Activity Diagram, State Chart diagram, Communication Diagram) here to outline the dynamic nature of the PROBLEM 
```

Comment:
Not sure I see the reason to lowercase "diagram" when it is capitalized in the other two names.